projects/addon-doc/components/changelog/changelog.component.ts

```diff
@@ -0,0 +1,78 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Inject,
+    Input,
+    Optional,
+    Sanitizer,
+    SecurityContext,
+} from '@angular/core';
+import {DomSanitizer, SafeHtml} from '@angular/platform-browser';
+import {TuiRawLoaderContent} from '@taiga-ui/addon-doc/interfaces';
+import {tuiRawLoad} from '@taiga-ui/addon-doc/utils';
+import {TuiSafeHtml} from '@taiga-ui/cdk';
+import {TUI_SANITIZER} from '@taiga-ui/core';
+import {marked, Renderer} from 'marked';
+import {map, Observable, of, switchMap} from 'rxjs';
+
+@Component({
+    selector: 'tui-changelog',
+    templateUrl: './changelog.component.html',
+    styleUrls: ['./changelog.component.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TuiChangelogComponent {
+    value: Observable<TuiSafeHtml> = of('');
```

Comment:
Usually we have subject to next to and an observable piped from it, not a complete observable creation inside the setter.

Reply:
fixed